packages/mdc-tabs/_mixins.scss

```diff
@@ -0,0 +1,88 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import "@material/theme/mixins";
+@import "../variables";
+
+@mixin mdc-tab-label-ink-color($color) {
+  .mdc-tab:not(.mdc-tab--active) {
```

Comment:
I go back and forth on the best way to support states within an element. In this case its how to support customization of both the inactive and active state of a tab.

In mdc-button, we did *not* want to allow customization of the disabled button, so we added a `:not(:disabled)` to our public mixins.

However, in this mdc-tab case, I think we want to support customization of active and inactive tabs separately.

I propose this:
* one public sass mixins `mdc-tab-label-ink-color`
* drop `mdc-tab-label-activated-ink-color`
* change mdc-tab-bar.scss to have:

```
.mdc-tab {
  @include mdc-tab-label-ink-color(text-secondary-on-light);
}
.mdc-tab--active {
  @include mdc-tab-label-ink-color(text-primary-on-light);
}
```

Then clients should override the the active inactive state like this
```
.foo-tab:not(.mdc-tab--active) {
  @include mdc-tab-label-ink-color(pink);
}
.foo-tab.mdc-tab--active {
  @include mdc-tab-label-ink-color(purple);
}
```

I think it is more important that clients keep their active and inactive states in sync, then it is for clients to keep their subcomponents in sync. For example this makes a lot of sense:
```
.foo-tab:not(.mdc-tab--active) {
  @include mdc-tab-label-ink-color(pink);
  @include mdc-tab-icon-ink-color(pink);
}
.foo-tab.mdc-tab--active {
  @include mdc-tab-label-ink-color(purple);
  @include mdc-tab-icon-ink-color(purple);
}
```

But this is harder to read
```
.foo-tab {
  @include mdc-tab-label-ink-color(pink /*inactive color*/, purple /*active color*/);
  @include mdc-tab-icon-ink-color(pink /*inactive color*/, purple /*active color*/);
}
```

Reply:
If we go that route won't have mixins on the parent class, which I thought was what we wanted. Also the label-ink-color mixin would be a proxy to mdc-theme-prop:

```
@mixin mdc-tab-label-ink-color($color) {
  @include mdc-theme-prop(color, $color);
}
```

The icon would also just be:
```
@mixin mdc-tab-icon-ink-color($color) {
  @include mdc-theme-prop(color, $color);
}
```

What is the main purpose for these scss mixins? Is it to allow clients to customize the components quicker? To make our styles DRY'er? In either case if our scss mixins for other components are designed to live on the parent class, we should try to be consistent here as well. Clients and external developers can easily see the mixins we've created and know where to put them for quick configuration. 

I'll probably pose the question on theming since you are on vacayy ;)